pkg/ride/functions_proto.go

```diff
@@ -163,6 +163,43 @@ func invoke(env Environment, args ...rideType) (rideType, error) {
 	return nil, errors.Errorf("result of Invoke is false")
 }
 
+func hashScriptAtAddress(env Environment, args ...rideType) (rideType, error) {
+	recipient, err := extractRecipient(args[0])
+	if err != nil {
+		return nil, errors.Errorf("unexpected argument type '%s'", args[0].instanceOf())
```

Comment:
Please, put function name in error messages, it's really helpful during investigation of script failures.